applications/app/services/NewspaperQuery.scala

```diff
@@ -18,32 +18,50 @@ case class ContentByPage(page: Int, content: ApiContent)
 case class TagWithContent(tag: Tag, content: ApiContent)
 case class BookSectionContentByPage(page: Int, booksectionContent: BookSectionContent)
 
-object TodaysNewspaperQuery extends ExecutionContexts with Dates with Logging {
+object NewspaperQuery extends ExecutionContexts with Dates with Logging {
 
   val dateForFrontPagePattern = DateTimeFormat.forPattern("EEEE d MMMM y")
-  def fetchTodaysPaper: Future[List[FaciaContainer]] = {
-    val today = {
-      val now = DateTime.now(DateTimeZone.UTC)
-      if(now.getDayOfWeek() == DateTimeConstants.SUNDAY) now.minusDays(1) else now
-    }
+  val FRONT_PAGE_DISPLAY_NAME = "front page"
+
+  def fetchTodaysPaper: Future[List[FaciaContainer]] =
+    bookSectionContainers(DateTime.now(DateTimeZone.UTC))
+
+
+  def fetchPaperForDate(day: String, month: String, year: String): Future[List[FaciaContainer]] = {
+    val dateFormatUTC = DateTimeFormat.forPattern("yyyy/MMM/dd").withZone(DateTimeZone.UTC)
+
+    val date = dateFormatUTC
+      .parseDateTime(s"$year/$month/$day")
+      .withTimeAtStartOfDay()
+      .plusDays(1)
```

Comment:
Watch this bit. We're setting the date to today at 23:59:59 which is fine if it's used as a to-date, but maybe not as a from-date.